src/coreclr-debug/activate.ts

```diff
@@ -37,27 +37,32 @@ export async function activate(thisExtension: vscode.Extension<CSharpExtensionEx
     context.subscriptions.push(vscode.debug.registerDebugAdapterDescriptorFactory('clr', factory));
 }
 
-async function checkForInvalidArchitecture(platformInformation: PlatformInformation, eventStream: EventStream): Promise<boolean> {
+async function checkIsValidArchitecture(platformInformation: PlatformInformation, eventStream: EventStream): Promise<boolean> {
     if (platformInformation) {
         if (platformInformation.isMacOS()) {
             if (platformInformation.architecture === "arm64") {
                 eventStream.post(new DebuggerPrerequisiteWarning(`[WARNING]: arm64 macOS is not officially supported by the .NET Core debugger. You may experience unexpected issues when running in this configuration.`));
-                return false;
+                return true;
             }
 
             // Validate we are on compatiable macOS version if we are x86_64
             if ((platformInformation.architecture !== "x86_64") || 
                 (platformInformation.architecture === "x86_64" && !CoreClrDebugUtil.isMacOSSupported())) {
                 eventStream.post(new DebuggerPrerequisiteFailure("[ERROR] The debugger cannot be installed. The debugger requires macOS 10.12 (Sierra) or newer."));
-                return true;
+                return false;
             }
+
+            return true;
         }
-        else if (platformInformation.architecture !== "x86_64") {
-            if (platformInformation.isWindows() && platformInformation.architecture === "x86") {
+        else if (platformInformation.isWindows()) {
+            if (platformInformation.architecture === "x86") {
                 eventStream.post(new DebuggerPrerequisiteWarning(`[WARNING]: x86 Windows is not currently supported by the .NET Core debugger. Debugging will not be available.`));
-            } else {
-                eventStream.post(new DebuggerPrerequisiteWarning(`[WARNING]: Processor architecture '${platformInformation.architecture}' is not currently supported by the .NET Core debugger. Debugging will not be available.`));
+                return false;
             }
+
+            return true;
+        }
+        else {
             return true;
```

Comment:
This enables support for Linux arm/arm64. This works for the debugger, but I am not sure we want to do this since the language service doesn't work. @JoeRobich do you have an opinion?

Reply:
Oh, I am sorry, this is just debugging. Nevermind.